### ISSUE TITLE
Ecs update-service does not work with ARN

### ISSUE THREAD

**Issue by rifqifatih** [spulec/moto#2958]
Hi, 

I tried updating ecs by using serviceArn as parameter for service and resulting in `An error occurred (ServiceNotFoundException) when calling the UpdateService operation: The service arn:aws:ecs:us-east-1:012345678910:service/test_ecs_service does not exist`.

IIRC updating with arn should be possible in boto3 / awscli. Here's the test to reproduce:

```
@mock_ecs
def test_update_service_with_arn():
    client = boto3.client("ecs", region_name="us-east-1")
    _ = client.create_cluster(clusterName="test_ecs_cluster")
    _ = client.register_task_definition(
        family="test_ecs_task",
        containerDefinitions=[
            {
                "name": "hello_world",
                "image": "docker/hello-world:latest",
                "cpu": 1024,
                "memory": 400,
                "essential": True,
                "environment": [
                    {"name": "AWS_ACCESS_KEY_ID", "value": "SOME_ACCESS_KEY"}
                ],
                "logConfiguration": {"logDriver": "json-file"},
            }
        ],
    )
    response = client.create_service(
        cluster="test_ecs_cluster",
        serviceName="test_ecs_service",
        taskDefinition="test_ecs_task",
        desiredCount=2,
    )
    response["service"]["desiredCount"].should.equal(2)
    response["service"]["serviceArn"].should.equal(
        "arn:aws:ecs:us-east-1:012345678910:service/test_ecs_service"
    )
    response = client.update_service(
        cluster="test_ecs_cluster",
        service="arn:aws:ecs:us-east-1:012345678910:service/test_ecs_service",
        taskDefinition="test_ecs_task",
        desiredCount=0,
    )
    response["service"]["desiredCount"].should.equal(0)
    response["service"]["schedulingStrategy"].should.equal("REPLICA")
```

**Comment by bblommers** [comment]
Can you verify that is the case against a real AWS instance?
All the [documentation](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_UpdateService.html) only talks about the `name of the service` - for parameters where it accepts the name _or ARN_ it's always explicitly mentioned. 

This could be just the docs being out of date, of course. :)

**Comment by rifqifatih** [comment]
Yes, confirmed by running `aws ecs update-service --cluster $CLUSTER_NAME --service $SERVICE_ARN --desired-count 1`

Also the case with `describe-services` which the [documentation](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_DescribeServices.html) doesn't explicitly stated to accept ARN. But Moto already implemented for this function this.

**Comment by bblommers** [comment]
This should be fixed as of moto >= 1.3.15.dev999
